packages/plugin-core/assets/dendron-ws/tutorial/treatments/main/tutorial.taking-notes.md

```diff
@@ -0,0 +1,143 @@
+
+![Children](https://org-dendron-public-assets.s3.amazonaws.com/images/children.png)
+
+### Create a hierarchy at any level
+
+You can create a hierarchy at any level. For example, type `recipes.italian.desserts.tiramisu` in the lookup bar and press **Enter**.
+
+You'll see there is a `+` sign next to `italian` and `desserts` when viewing in the lookup or tree view. The plus sign indicates that this note is a stub. A **[stub](https://wiki.dendron.so/notes/c6fd6bc4-7f75-4cbb-8f34-f7b99bfe2d50)** is a placeholder for a note that hasn't been created. Dendron uses stubs to avoid cluttering your file system with empty notes when creating hierarchies.
+
+![Tree View Dark](https://org-dendron-public-assets.s3.amazonaws.com/images/tutorial-tree-view-dark.png)
+
+Following this recipes example, you can create other related notes. Some examples:
+
+```
+recipes.favorites
+recipes.italian.appetizers
+recipes.chinese
+recipes.chinese.appetizers.wontons
+recipes.ingredients.favorites
+recipes.ingredients.shopping-list
+```
+
+Read [Why Dendron uses hierarchies](https://wiki.dendron.so/notes/683740e3-70ce-4a47-a1f4-1f140e80b558) for detailed information on Dendron note structure. Dendron also allows you to change the hierarchy at a later date. Read [Refactoring in Dendron](https://wiki.dendron.so/notes/9MZBqhrijEM4QpZRa5t08) in case you change your mind on how to organize your notes.
+
+## Find your Notes
+
+To find your notes, do the following:
+
+1. Enter `Ctrl+L` / `Cmd+L` to open the lookup bar.
+2. Enter the name of your note. For example, you can type `vege` to find your `recipes.vegetarian` note.
+3. Press **Enter**. The note is opened.
+
+> 💡 **INFO:** Lookup uses _fuzzy search_, which means you can type out partial queries and still see the results.Entering multiple keywords delimited by space will lookup matching notes regardless of order of the > keywords.
+>
+> - Example: `vege rec` or `rec vege` will match the `recipes.vegetarian` note.
+
+> 💡 **TIP:** Read [Finding Notes](https://wiki.dendron.so/notes/84a0366a-eab5-4862-9bef-360f92a196dc) to understand the different ways of querying the lookup.
+> Most queries can be satisfied using [lookup](https://wiki.dendron.so/notes/84a0366a-eab5-4862-9bef-360f92a196dc), but sometimes you want to do a full text search across the **contents** of your notes. This can be done with `Search: Find in Files` (`Ctrl+Shift+F` / `Cmd+Shift+F`), which leverages VS Code [built-in search](https://code.visualstudio.com/docs/editor/codebasics#_search-across-files) capabilities.
+
+Dendron also has a few built-in note types. For example, if you find yourself taking meeting notes often, you can use the `Dendron: Create Meeting Note` command to create a note with a pre-built template for meetings. Try it out!
```

Comment:
for parity (and experimental consistency), can we also add a level 3 header for the meeting notes section? 

```markdown
### Meeting Notes

Dendron also has a few built-in note types...
```

Reply:
oops. missed that from the constant.

this already got merged so I raised another PR to sync this: https://github.com/dendronhq/dendron/pull/2919
PR for `dendron-site` already has it included.